e2e-jar-test/fips/src/test/java/net/snowflake/FipsIngestE2ETest.java

```diff
@@ -33,7 +33,6 @@ public void basicTest() throws InterruptedException {
   }
 
   @Test
-  @Ignore("Takes too long to run")
```

Comment:
add a comment in case you forget to add it back

Reply:
I will remove it in next commit!